test/components/New_Order_spec.js

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import React from 'react';
+import chai from 'chai';
```

Comment:
Oh I just meant combine with line 1: `import { expect, spy } from 'chai';`

Then don't need to call `spy()` off `chai` either.